test/examples/causal_consistency.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const setupDatabase = require('../functional/shared').setupDatabase;
+const expect = require('chai').expect;
+const MongoClient = require('../../lib/mongo_client');
+
+describe('examples(causal-consistency):', function() {
+  let client;
+  let collection;
+  let session;
+
+  before(async function() {
+    await setupDatabase(this.configuration);
+  });
+
+  beforeEach(async function() {
+    client = await MongoClient.connect(this.configuration.url());
+    collection = client.db(this.configuration.db).collection('arrayFilterUpdateExample');
+  });
+
+  afterEach(async function() {
+    if (session) {
+      await session.endSession();
+      session = undefined;
+    }
+
+    await client.close();
+    client = undefined;
+    collection = undefined;
+  });
+
+  it('CausalConsistency', {
```

Comment:
seems a little inconsistent (pun intended), maybe `it('supports causal consistency', ...)` ?